data-files/samples/targets.Status.Any

```diff
@@ -0,0 +1,12 @@
+{ 
+    currentUser = "Sample User"; 
+    sessions = ( "1target", "2targets", "5targets", "moving", "jumping", "mixedtypes"); 
+    settingsVersion = 1; 
+    users = ( 
+        { 
+            completedSessions = ( ); 
+            id = "Sample User"; 
+            sessions = ( "1target", "2targets", "5targets", "moving", "jumping", "mixedtypes"); 
```

Comment:
Specifying this and the `sessions` array above in the same order is a bit odd, perhaps include a comment to describe why. Generally speaking I usually just spec on or the other.

Reply:
The one above is in case an experiment ends up allowing adding new users. I'm not sure it's all that important, and the functionality is documented in the user status documentation.